zeek/assets/logs/zeek_tests.yaml

```diff
@@ -0,0 +1,203 @@
+id: zeek
```

Comment:
@txominpelu This first sample is the one we'ee having difficulty getting to work.

From Mohit @ Crest: can you please help us understand where should ddtags attributes go in samples YAML?

I've tried several ways to specify ddtags in samples but none of them seems to work. For example,

1. ddtags as individual attribute
Result: service query pipelines were not applied
```
- sample: |-
    {...}
  ddtags: ["service:authentication"]
  results: null
  ```
  
2. breaking down sample into two attributes message and ddtags
Error: internal error on CI check
```
- sample:
    message: |-
      {...}
    ddtags: ["service:authentication"]
  results: null
  ```
  
3. modified sample and made ddtags part of sample json
Result: service query pipelines were not applied
```
- sample: |-
    {
      ...,
      "ddtags": ["service:authentication"]
    }
  results: null
  ```

Reply:
Hi @txominpelu, If you want to check the different approaches I have tried then you can find them in PR https://github.com/DataDog/integrations-core/pull/16746. I've made several commits trying few ways to add ddtags but I can't get it to work.

Reply:
@mohittilala-crest the logs folks just rolled out a new version of the CI job that will support this format:

```
{
       "service" : "corelight",
       "message" : "<134>Dec 26 01:35:11 machine-name {\"_path\":\"capture_loss\",\"_write_ts\":\"2023-12-12T05:52:50.756358Z\",\"ts\":\"2023-12-12T05:52:32.763303Z\",\"ts_delta\":15.235642194747925,\"peer\":       \"zeek\",\"gaps\":3,\"acks\":316,\"percent_lost\":0.9493670886075949}"
}
```

please give it a shot and let me know how it goes. Thanks!